### PR TITLE
Add instructions to add your own themes.

### PR DESCRIPTION
**Pull request by aayushdutt**
I have tested this method and it works flawlessly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,3 @@ content/data/*.db
 content/logs/
 config.production.json
 content/data/ghost-local.db
-
-content/themes/*
```
Comment (JaneJeon) [inline .gitignore:9]: The `content/themes/` need to be excluded because they're meant to be copied over from `node_modules`, NOT hardcoded in the repo.

Revert this change.

File: README.md [hunk 2]
```diff
@@ -106,20 +106,35 @@ This repository is a [Node.js](https://nodejs.org) web application that specifie
...
 
+
+## Using a new theme
+
+Due to the Ephimeral file system of Heroku, currently the admin panel doesn't support uploading the themes directly. If uploaded, the theme would go missing after some time. The only way to use custom theme currently is to commit your themes in the code files and then re publish the application.
+
+**To use a new theme follow these steps**
+
+1. Fork and clone this repository
+2. Extract and copy your theme folder in content/themes. If your theme is `Editorial` it should look like `content/themes/Editorial/...`
```
Comment (JaneJeon) [inline README.md:117]: Again, no. The correct way to do it is to host your own theme on github/gitlab/npm and then specify the versions in `package.json`. This entire section needs to be rewritten.

File: README.md [hunk 3]
```diff
@@ -106,20 +106,35 @@ This repository is a [Node.js](https://nodejs.org) web application that specifie
...
+In case you wish to **update existing Heroku deployment** with a new theme, follow steps 1 - 3 and then do the steps mentioned in [Updating the source code](https://github.com/vyronarediano/slp-on-heroku#updating-source-code)
+
 ## Updating source code
 
-Optionally after deployment, to push Ghost upgrades or work with source code, clone this repo (or a fork) and connect it with the Heroku app:
+Optionally after deployment, to push Ghost upgrades or work with source code, fork this repository and make required changes.
 
+**Connect the app to your Heroku deployment:** 
 ```bash
-git clone https://github.com/snathjr/ghost-on-heroku
```
Comment (JaneJeon) [inline README.md:114]: This shouldn't really be removed.

File: README.md [hunk 4]
```diff
@@ -106,20 +106,35 @@ This repository is a [Node.js](https://nodejs.org) web application that specifie
...
-cd ghost-on-heroku
-
 heroku git:remote -a YOURAPPNAME
 heroku info
 ```
 
 Then you can push commits to the Heroku app, triggering new deployments:
-
```
Comment (JaneJeon) [inline README.md:122]: Nitpick?

File: README.md [hunk 5]
```diff
@@ -106,20 +106,35 @@ This repository is a [Node.js](https://nodejs.org) web application that specifie
...
+Due to the Ephimeral file system of Heroku, currently the admin panel doesn't support uploading the themes directly. If uploaded, the theme would go missing after some time. The only way to use custom theme currently is to commit your themes in the code files and then re publish the application.
+
+**To use a new theme follow these steps**
+
+1. Fork and clone this repository
+2. Extract and copy your theme folder in content/themes. If your theme is `Editorial` it should look like `content/themes/Editorial/...`
+3. Push the change
+4. Deploy to Heroku by clicking the button below
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (JaneJeon) [inline README.md:121]: I wish you would at least look through the closed issues list before making changes on your own. See #32 for why the querystring is necessary. Revert this change.

File: README.md (hunk 2 above)
Comment (aayushdutt) [reply README.md:117]: @JaneJeon I can see your point there but I still think this is a better way to host themes. Here are my reasons:
1. The method of hosting my theme on GitHub and specifying versions in `package.json` has way too much complexity. The things you have to do to get it working are:
    * Create a new repository and push your theme there.
    * Fork and clone this repository
    * Add your theme in `package.json`
    * Add your theme name in `copy-themes.sh`
    * Push your changes to fork and deploy.

Compared to that, the method I am suggesting has fewer moving points:
    * Fork and clone this repository
    * Copy over your theme to `content/themes/`
    * Push your changes and deploy
After all this repository aims to reduce work from the user, we want the work to be minimal.

2. While I can see your method work great for open source themes (the fact that it will fetch the latest theme post-installation), it doesn't work at all if a user wishes to install a paid theme. The user won't want to host their theme on a public GitHub repository and currently, there is no way to fetch it from a private one.

3. I can clearly see why your method makes sense for open source themes. We don't have to bloat the repository with extra stuff. But if the user wishes to add one extra theme, copying over shouldn't be a problem.

File: README.md (hunk 5 above)
Comment (aayushdutt) [reply README.md:121]: Got it, I'll change that

File: README.md (hunk 3 above)
Comment (aayushdutt) [reply README.md:114]: I don't really see a point in cloning this repository, making a change in it and not tracking that change. The correct way to change the source code should always be to `fork`

File: README.md (hunk 4 above)
Comment (aayushdutt) [reply README.md:122]: As the heading of this section is: "Connect the app to your Heroku deployment", it makes sense to remove the cd step

File: .gitignore (hunk 1 above)
Comment (aayushdutt) [reply .gitignore:9]: @JaneJeon I don't think this should be a problem. See my points on why pushing your themes to this repository might be a better idea. Also, since the included themes are copied over to content/themes after the install step (which happens only on heroku), we should not have to worry about getting them there in `content/themes` in the source code

File: README.md (hunk 2 above)
Comment (JaneJeon) [reply README.md:117]: You can use private packages/verdaccio and such to get around "non-open-source" themes. My main concern is that it will clutter up your repository with the *actual* themes checked into the codebase, not version-tagged or anything like that. Good luck updating your themes, because you'll still need to do the "way too complex" steps of pinning your theme in `package.json` and in the script. It's LITERALLY two lines.

File: .gitignore (hunk 1 above)
Comment (JaneJeon) [reply .gitignore:9]: Of *course* all the themes belong in `content/themes`, as specified by Ghost CMS.

The question is whether you want to check the raw theme code into your codebase, and that's really just a hacky AF solution. Come up with a better way (that doesn't involve checking in individual themes) or remove this change entirely.

File: README.md (hunk 2 above)
Comment (aayushdutt) [reply README.md:117]: Updating the theme should be equal work. It only involves copying the updated theme and redeploying. That is equivalent if to editing the package.json and redeploying.

> You can use private packages/verdaccio and such to get around "non-open-source" themes

I don't think this is a good solution.

Comment (aayushdutt) [reply README.md:117]: @JaneJeon  I can't find a better way to easily add (private) themes without managing multiple repositories and editing a bunch of files. Copying over and pushing is the quickest and easiest to go.

If you think this change won't improve the project, feel free to close this PR.
A quick note though, it will be really good to value an open-source contributors' efforts and time a little more. Having an open-minded and polite discussion always helps.

**Comment by JaneJeon** [comment]
@aayushdutt I have a few days of free time to work on open source stuff. Is your PR ready for review? If so, resolve all discussions and I'll look through it again.
